brainframe/api/bf_codecs/condition_codecs.py

```diff
@@ -109,7 +108,7 @@ def from_dict(d: dict):
         with_attribute = None
         if d["with_attribute"] is not None:
             with_attribute = Attribute.from_dict(d["with_attribute"])
-        test = CountConditionTestType(d["test"])
+        test = ZoneAlarmCountCondition.TestType(d["test"])
```

Comment:
something I've never been able to decide is if it's better to do `ZoneAlarmCountCondition.TestType` or `self.TestType` in situations like this. Doing the former means that you can't subclass and override the `TestType` Enum, but doing the latter might make it confusing to understand what `TestType` is